### PR TITLE
Add microk8s to server download thankyou page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Added a card for microk8s on the /download/server/thank-you page
- Added some bottom margin to the recaptcha box

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/server/thank-you
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc]()

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1542

## Screenshots

image

### CODE REVIEW

File: templates/download/server/thank-you.html [hunk 1]
```diff
@@ -36,6 +36,33 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Server</h1>
     </div>
   </section>
 
+  <section class="p-strip--light is-shallow u-no-">
```
Comment (anthonydillon) [inline templates/download/server/thank-you.html]: Missing the end of the utility class

File: templates/download/server/thank-you.html [hunk 2]
```diff
@@ -36,6 +36,33 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Server</h1>
...
+        <h4>
+          Lightweight, reliable Kubernetes
+        </h4>
+        <div class="row">
+          <div class="col-4">
+            <p>
+              Need a fast, reliable, single-node K8s up and running under 60 seconds?  Made for developers, great for IoT and AI on the edge.
+            </p>
+            <p>
+              <a class="p-button is-wide" href="https://microk8s.io/">
```
Comment (anthonydillon) [inline templates/download/server/thank-you.html]: I think this button could be with no margin bottom class

**Comment by anasereijo** [comment]
As we can't show Multipass yet I think it looks odd having that white space on the right side. I suggest creating a full-width strip and later having 2 cards (one for microk8s other for multipass).

I designed 2 versions. @kwm14 could you please let me know your thoughts? 

### Version 1
server thank you 1


### Version 2
server thank you 2

**Comment by kwm14** [comment]
@anasereijo Design V1 LGTM 👍

**Comment by anasereijo** [comment]
@kwm14 that's my preferred version too, thanks!
@pmahnke could you please update the page to match the design? Any questions let me know

**Comment by pmahnke** [comment]
@anasereijo which do you like better?	

this is the col-2 150px version
image


this is the col-1 100px version - my perfered
image

**Comment by caldav** [comment]
@pmahnke Would it be possible to use it for the AB test and wait for results before implementation?

**Comment by anasereijo** [comment]
@pmahnke 100px version is my favourite too! looks great

**Comment by pmahnke** [comment]
@caldav, what ab test?  did you want to use this design for it?  can we see what you are testing?

**Comment by caldav** [comment]
@pmahnke Test would be this against original. Measuring impact on ebook downloads VS impact on microk8s growth (CTR and install). I guess the question to answer is: is it worth pushing down the form? It might give MicroK8s exposure, but may not drive much installs because the OS will still be downloading/installing when people are visiting microk8s.io.

File: templates/download/server/thank-you.html [hunk 3]
```diff
@@ -36,6 +36,32 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Server</h1>
...
+        <h4>
+          Lightweight, reliable Kubernetes
+        </h4>
+        <p>
+          Need a fast, reliable, single-node K8s up and running under 60 seconds?
+          <br />
+          Made for developers, great for IoT and AI on the edge.
+        </p>
+        <p>
+          <a class="" href="https://microk8s.io/">
```
Comment (anthonydillon) [inline templates/download/server/thank-you.html]: empty class

**Pull request merged**